README.md

```diff
@@ -62,6 +62,7 @@ SIMD is supported on `x86`, `x86_64` and `wasm32` targets.
   `RUSTCFLAGS`.
 * To enable `simd128` on `wasm32` targets add `-C target-feature=+simd128` to
   `RUSTFLAGS`.
+* To enable `NEON` on `aarch64` targets add `-C target-feature=+neon` to `RUSTFLAGS`.
```

Comment:
This is a bit less clear to me on aarch64. For the platforms I tested on which were raspberry pi 4 and Mac M1,  NEON was enabled by default.  For now it's probably better to copy the x86_64 line and say it's on by default.

Reply:
Got it! I searched online, and it looks like `NEON` is required for `aarch64` processors.